MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -559,7 +559,7 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
                                          }];
     }
 
-    if ([details.releaseNotes length] > 0) {
+    if ([details.releaseNotes length] > 0 && details.releaseNotesUrl) {
```

Comment:
Does the release note will still be part of the release details. If not then this part will be skipped. Shouldn't we remove the details.releaseNotes?